src/components/date.js

```diff
@@ -28,7 +28,7 @@ export default class FormattedDate extends Component {
             return children(formattedDate);
         }
 
-        return <span>{formattedDate}</span>;
+        return <time datetime={value}>{formattedDate}</time>;
```

Comment:
<date> is not a valid HTML5 tag?

And yes that is true is there a parsed format of the input date we can easily get to?